app/src/main/java/com/money/manager/ex/transactions/EditTransactionCommonFunctions.java

```diff
@@ -605,7 +605,7 @@ public void onClick(DialogInterface dialogInterface, int i, boolean b) {
                 }
             });
 
-            builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
+            builder.setPositiveButton(R.string.OK, new DialogInterface.OnClickListener() {
```

Comment:
would we just use some built-in ones for the common strings? e.g. android.R.string.ok android.R.string.cancel